interface/forms/newpatient/common.php

```diff
@@ -437,8 +434,7 @@ function cancelClickedOld() {
                         <?php
                         if (!$GLOBALS['gbl_visit_referral_source']) {
                             echo "style='display:none' ";
-                        } ?>
-                    >
+                        } ?>>
                         <div class="col-sm-2">
                             <label for="form_referral_source" class="oe-text-to-right"><?php echo xlt('Referral Source'); ?>:</label>
                         </div>
```

Comment:
Hi @bradymiller  and @tywrenn 

Such as this `oe-text-to-right` class, Is it can convert to bootstrap `text-right` ?

Reply:
@stu01509 Yep the `oe-text-to-right` class is the same as `text-right`. Also the same for left. You can go ahead and replace those with BS4 classes instead.

Reply:
Not really sure about the other oe classes as they have some differences yet I know for sure these ones can be replaced.